.travis.yml

```diff
@@ -37,21 +37,63 @@ addons:
       - g++-4.8
 
 before_script:
-  - du -sh ./node_modules ./bower_components/ ./docs/bower_components/ || true
-  - ./scripts/travis/before_build.sh
+  - du -sh ./node_modules ./bower_components/ || true
+  - "./scripts/travis/before_build.sh"
 
 script:
-  - ./scripts/travis/build.sh
+  - "./scripts/travis/build.sh"
 
 after_script:
-  - ./scripts/travis/tear_down_browser_provider.sh
-  - ./scripts/travis/print_logs.sh
+  - "./scripts/travis/tear_down_browser_provider.sh"
+  - "./scripts/travis/print_logs.sh"
 
 notifications:
   webhooks:
     urls:
       - https://webhooks.gitter.im/e/d2120f3f2bb39a4531b2
       - http://104.197.9.155:8484/hubot/travis/activity #hubot-server
-    on_success: always  # options: [always|never|change] default: always
-    on_failure: always  # options: [always|never|change] default: always
-    on_start: always    # default: false
+    on_success: always # options: [always|never|change] default: always
+    on_failure: always # options: [always|never|change] default: always
+    on_start: always # default: false
+
+jobs:
+  include:
+    - stage: deploy
+      env:
+        - JOB=deploy
+      before_script: skip
+      script:
+        - "./scripts/travis/build.sh"
```

Comment:
Not directly related to this PR, but I noticed that the build script is run even on PRs (which will never get deployed).
Could we skip the extra work on PRs?